build.fsx

```diff
@@ -126,28 +126,11 @@ Target "CopyOutput" (fun _ ->
 )
 
 //--------------------------------------------------------------------------------
-// NBench targets 
+// Benchmarks
 //--------------------------------------------------------------------------------
 
-Target "NBench" (fun _ ->
-    if (isWindows) then
-        let nbenchTestPath = findToolInSubPath "NBench.Runner.exe" "tools/NBench.Runner/lib/net45"
-        let assembly = __SOURCE_DIRECTORY__ @@ "Hyperion.Tests.Performance/bin/Release/net45/Hyperion.Tests.Performance.dll"
-        
-        let spec = getBuildParam "spec"
-
-        let args = new StringBuilder()
-                |> append assembly
-                |> append (sprintf "output-directory=\"%s\"" outputPerfTests)
-                |> append (sprintf "concurrent=\"%b\"" true)
-                |> append (sprintf "trace=\"%b\"" true)
-                |> toText
-
-        let result = ExecProcess(fun info -> 
-            info.FileName <- nbenchTestPath
-            info.WorkingDirectory <- (Path.GetDirectoryName (FullName nbenchTestPath))
-            info.Arguments <- args) (System.TimeSpan.FromMinutes 15.0) (* Reasonably long-running task. *)
-        if result <> 0 then failwithf "NBench.Runner failed. %s %s" nbenchTestPath args
+Target "Benchmarks" (fun _ ->
+    () //TODO: complete BenchmarkDotNet setup
```

Comment:
I will need some help here - in general BenchmarkDotNet is run as console application. Right now I'm using default output paths for benchmark results, but they could be possibly passed as cmd args.